hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -776,7 +776,7 @@
       hwaccess: "hro",
       regwen:   "REGWEN",
       fields: [
-        { bits: "3:0",
+        { bits: "31:0",
```

Comment:
this is not strictly related to this PR, but we should probably make a note to revisit.
The security labs will probably argue that if all 0's disable the alert, it might become an attack point to glitch this register.

So we will perhaps need to think about how to protect this value against that.

One argument could be that we make this actually 16b wide only, and for the upper copy to be complement of the latter.
Or we could also think about making this a shadowed register.

@cdgori what do you think?

Reply:
Yes, that's likely to be pointed out (and is a real concern since it would disable the alert).

If we think 16-bit is enough to hold the value (should be, I think?) then upper half as complement is good enough for me to solve the issue.